activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/Application.java

```diff
@@ -28,8 +31,25 @@
         "org.activiti.cloud.services.test.identity.keycloak.interceptor"})
 public class Application {
 
+    static GenericContainer keycloakContainer = new GenericContainer("activiti/activiti-keycloak")
+            .withExposedPorts(8180)
+            .waitingFor(Wait.defaultWaitStrategy());
+
+    static RabbitMQContainer rabbitMQContainer = new RabbitMQContainer("rabbitmq:management");
+
+    static {
+        keycloakContainer.start();
```

Comment:
you may want to start both containers in parallel, e.g. with:
```
Startables.deepStart(Stream.of(keycloakContainer, rabbitMQContainer)).join();
```

Reply:
WOW, thanks for the suggestion